1904-11-03.xml

```diff
@@ -0,0 +1,2652 @@
+                        from their respective standpoints. It was the Russian late effort to march
+                        south, to relieve Port Arthur, which influenced the Japanese commander to
+                        replace offensive strategy by defensive. Similarly, it was the Japanese
+                        impetuous northerly advance that constrained Kuropatkin, after the failure
+                        of his attempt to get south, to halt on the Sha-ho. In the one case, the
+                        object is the capture of Port Arthur by starvation and storm combined before
+                        relief arrives; in the other, the Russians feel them- selves bound to retain
+                        possession of Mukden. at any cost. In effect, it is a game of patience, and
+                        it only remains to be seen which side will be the first to give way to
+                        irritation at the delay </p>
+                </div>
+            </div>
+
+            <pb n="3"/>
+            <!-- missing 3 items -->
+            <div type="item">
+                <head>THE KHEDIVE </head>
+                <p>The Khedive inspected his estates yesterday at Mariout, and returned in the
+                    evening to .Montazah palace, where he is spending today. The Gonvernorat of
+                    Cairo has issued invitations to numerous officials and notables to present
+                    themselves at the station at 11.30 a.m. next Tuesday to greet hisHighness on his
+                    arrival. The Ministry of War has given orders for a detachment to be' present as
+                    a guard of () The train in which his Highness travels will stop at Damanhour,
+                    Tantah, (Buket el) San, and Benha. </p>
+            </div>
+            <div type="advert">
+                <head>WINDSOR HOTEL. </head>
+                <p>Largest 1 class and most comfortable hotel in Alexandria. Facing the sea Central
+                    position. Under English management. 81.11.94 </p>
+            </div>
+            <div type="advert">
+                <head>BECK &amp;amp CO'S PILSENER BEER </head>
+                <p>BREMEN. </p>
+                <p>FEARS NO HONEST COMPETITION FOR QUALITY. </p>
+                <p>N.B.—Inferior Brande now being offered to Manager of certain good cercles. </p>
+                <p>Beware of evilly disposed competitors running down this very Superior Brand of
+                    Beer. </p>
+            </div>
+            <div type="advert">
+                <head>EDEN PALACE HOTEL, </head>
+                <p>CAIRO. </p>
+                <p>Open all the year round. </p>
+                <p>ELECTRIC LIGHT, —LIFT, —MODERATE CHARGES</p>
+            </div>
+            <div type="section">
+                <head> LOCAL AND GENERAL. </head>
+
+                <div type="item">
+                    <head/>
+                    <p>RAMADAN begins on Tuesday next.</p>
```

Comment:
add a </div> to the end of this line, so that the Ramadan item closes before the sleenan starts